services/minespace-web/package.json

```diff
@@ -29,6 +29,8 @@
     "@syncfusion/ej2-react-filemanager": "19.1.54",
     "@syncfusion/ej2-react-pdfviewer": "19.4.55",
     "@syncfusion/ej2-splitbuttons": "19.1.54",
+    "@types/jest": "^29.1.0",
```

Comment:
fix versions?

Reply:
Ah, good catch.  Yes, I started trying to update all of the jest stuff to 29, but couldn't get it to work, so rolled the others back to 24.  Will update this.